src/models/ITestStation.ts

```diff
@@ -9,6 +9,7 @@ export interface ITestStation {
   testStationTown: string;
   testStationAddress: string;
   testStationPostcode: string;
+  testStationCountry?: string;
```

Comment:
Shouldn't this field be required as it's now always going to be coming in from the dynamics feed?
```suggestion
  testStationCountry: string;
```